connect_to_db.ipynb

```diff
@@ -0,0 +1,232 @@
+{
```

Comment:
Could you add the python scripts taken out of the jupyter notebook for executing the sql script below in the repo?

Reply:
Since there is a lot of back and forth currently and we need to complete other parts of the project as well, I can fix these things later. For now lets just focus on getting main functionality done.

Reply:
will do that later. We are still changing a lot of things

Reply:
lgtm

Reply:
Deleted